app/src/main/java/com/sdp/movemeet/view/profile/EditProfileActivity.java

```diff
@@ -138,7 +150,7 @@ public void onFailure(@NonNull Exception e) {
 
 
     private void accessFirestoreUsersCollectionForUpdate() {
-        DocumentReference docRef = fStore.collection("users").document(user.getUid());
+        DocumentReference docRef = fStore.collection("users").document(userId);
```

Comment:
Did you use the Firestore abstraction somewhere and modify this kind of calls to the real Firebase Firestore service?